conf/app.conf

```diff
@@ -16,4 +16,4 @@ httpProxy = "127.0.0.1:10808"
 verificationCodeTimeout = 10
 initScore = 2000
 logPostOnly = true
-origin = "https://door.casbin.com"
\ No newline at end of file
+origin = "http://localhost:8000"

```

Comment:
Why change?

Reply:
This change is used to ensure that the docker image we build can be started without extra configuration, because docker images is built exactly according to the code in repo